### ISSUE TITLE
Can't get it going

### ISSUE THREAD

**Issue by grahamj** [20steps/cordova-plugin-watch-example#1]
Thanks for this, it looks great. I'm having trouble though - I followed the instructions and got everything seemingly working but the watch and phone apps don't seem to be communicating. I see changing values on the phone but only -1 on the watch.

I set my own team, changed the bundle identifiers and fixed a few problems that resulted from that, enabled groups on all targets, set the same group, entitlements look fine.

I can't think of what I might be missing, any thoughts perchance?

**Comment by dougnukem** [comment]
Did you setup an App Group and set the entitlements for the app and watch apps.

If you  don't have that setup the MMWormhole won't be able to send/receive values and you'll see `-1`

**Comment by grahamj** [comment]
Yep, did that. There was an Xcode update very recently, can you confirm this still works after that?
